Documentation/ContentObjects/Hmenu/Index.rst

```diff
@@ -1642,7 +1642,7 @@ generated from this array, which was returned from the function
                                ],
                                [
                                    'title' => 'New York Office',
-                                   '_OVERRIDE_HREF' => 'http://www.example.com',
+                                   '_OVERRIDE_HREF' => 'https://example.com',
```

Comment:
No `.org` here?

Reply:
The menu entry is explicitly overwriting the default domain, therefore i chose to use the next domain in the list.

Reply:
OK, thanks.